src/lms-ds-installer/mssql/0001-table-user.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE lms.[User] (

```

Comment:
This will eventually be replaced by output from MetaEd. Need something work with until then.